src/components/DashboardLayout/DashboardLayout.component.jsx

```diff
@@ -1,32 +1,48 @@
 import React from "react";
-import { Row, Col, Space } from "antd";
+import { Row, Col } from "antd";
 import DashboardSideBarComponent from "../DashboardNavigationSideBar/NavigationSideBar.component";
+import "./DashboardLayout.style.scss";
 
 const DashboardLayout = ({ navigationList, elementList, currentTab }) => {
   return (
     <>
-      <Row>
-        <Col>
+      <Row className="dashboard-menu-md">
+        <DashboardSideBarComponent
+          items={navigationList}
+          selectedKey={navigationList[currentTab].key}
+          mode="horizontal"
+        />
+      </Row>
+      <Row style={{ display: "flex", justifyContent: "center" }}>
+        <Col span={4} className="dashboard-menu-lg">
           <DashboardSideBarComponent
             items={navigationList}
             selectedKey={navigationList[currentTab].key}
+            mode="inline"
           />
         </Col>
-        <Col flex="auto">
-          <div
-            style={{
-              width: "100%",
-              display: "flex",
-              justifyContent: "center",
-              alignItems: "center",
-            }}
-          >
-            {elementList[currentTab]}
-          </div>
+        <Col flex="auto" sm={24} md={20} lg={20}>
+          <div>{elementList[currentTab]}</div>
         </Col>
       </Row>
+      <PatientMobileNavigation
```

Comment:
can you make this into reusable component ? like Mobilenavigation and then only need to pass the list and tab to work ? it's better than only patient tho

Reply:
actuallly I think we doesn't need it, I forgot to delete